scripts/ipxe-dts.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+error_exit() {
+    _error_msg="$1"
+    echo "$_error_msg"
+    exit 1
+}
+
+error_check() {
+    _error_code=$?
+    _error_msg="$1"
+    [ "$_error_code" -ne 0 ] && error_exit "$_error_msg : ($_error_code)"
+}
+
+userIP=$(ip route get 1 | sed 's/^.*src \([^ ]*\).*$/\1/;q')
+error_check "cannot get user IP"
+
+distroVersion=$(cat meta-dts/meta-dts-distro/conf/distro/dts-distro.conf | grep -F "DISTRO_VERSION" | awk -F'"' '{print $2}')
+error_check "cannot get distro version from 'meta-dts/meta-dts-distro/conf/distro/dts-distro.conf'"
+
+cd meta-dts/scripts/
+error_check "cannot cd to 'meta-dts/scripts'"
+
+mkdir -p ipxe-files
+error_check "cannot make directory 'ipxe-files'"
+
+cd ipxe-files
+error_check "cannot cd to 'ipxe-files'"
+
+wget --output-document=bzImage-v$distroVersion http://boot.dasharo.com/dts/v$distroVersion/bzImage-v$distroVersion
+error_check "cannot wget the bzImage v'$distroVersion' from 'http://boot.dasharo.com/dts/v$distroVersion/bzImage-v$distroVersion'"
+
+wget --output-document=dts-base-image-v$distroVersion.cpio.gz http://boot.dasharo.com/dts/v$distroVersion/dts-base-image-v$distroVersion.cpio.gz
+error_check "cannot wget the cpio.gz v'$distroVersion' from 'http://boot.dasharo.com/dts/v$distroVersion/dts-base-image-v$distroVersion.cpio.gz"
+
+if [ "$#"==0 ]; then
+
+    port=8000
+    python3 -m http.server $port &
+    error_check "failed to create http server with python on port:$port, ip:$userIP"
```

Comment:
```suggestion
    error_check "Failed to create HTTP server with python on port:$port, ip:$userIP."
```

Reply:
@TomaszAIR forgot to fix it fully in the last commit, so I had to do it [here](https://github.com/Dasharo/meta-dts/pull/96/commits/cad97aee1af3003095602c35f8de6de066ccd2c8)